gui/src/components/onboarding/pages/body-proportions/scaled-steps/ManualHeightStep.tsx

```diff
@@ -133,20 +142,7 @@ export function ManualHeightStep({
           >
             {l10n.getString('onboarding-automatic_proportions-prev_step')}
           </Button>
-          <Button
-            variant="primary"
-            onClick={() => {
-              const settingsRequest = new ChangeSettingsRequestT();
-              settingsRequest.modelSettings = new ModelSettingsT(
-                null,
-                null,
-                null,
-                new SkeletonHeightT(hmdHeight, 0)
-              );
-              sendRPCPacket(RpcMessage.ChangeSettingsRequest, settingsRequest);
-              nextStep();
-            }}
-          >
+          <Button variant="primary" onClick={handleSubmit(onSubmit)}>
```

Comment:
This is not the right way of doing it imo.
The form should wrap arround this button and have th onSubmit event call handle submit
Then the button can be set to type submit

Reply:
You can look at WifiCredsPage for an example